### ISSUE TITLE
errors in the last alpha. immediately after start-up

### ISSUE THREAD

**Issue by Nikita34196** [nvaccess/nvda#12323]
hi. in the last alpha, an error in the nvda.exe file appears immediately after startup. nvda actually doesn't work. there were also errors in the log. INFO - __main__ (12:28:28.821) - MainThread (8544):
Starting NVDA version alpha-22485,60c0b76b
INFO - core.main (12:28:28.914) - MainThread (8544):
Config dir: C:\Users\ntara\AppData\Roaming\nvda
INFO - config.ConfigManager._loadConfig (12:28:28.914) - MainThread (8544):
Loading config: C:\Users\ntara\AppData\Roaming\nvda\nvda.ini
INFO - core.main (12:28:29.266) - MainThread (8544):
Windows version: Windows 10 prerelease (10.0.21364) workstation
INFO - core.main (12:28:29.266) - MainThread (8544):
Using Python version 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:01:55) [MSC v.1900 32 bit (Intel)]
INFO - core.main (12:28:29.266) - MainThread (8544):
Using comtypes version 1.1.8
INFO - core.main (12:28:29.266) - MainThread (8544):
Using configobj version 5.1.0 with validate version 1.0.1
INFO - external:synthDrivers.RHVoice.SynthDriver.__init__ (12:28:29.638) - MainThread (8544):
Using RHVoice version b'1.2.0'
INFO - synthDriverHandler.setSynth (12:28:29.651) - MainThread (8544):
Loaded synthDriver RHVoice
INFO - core.main (12:28:29.651) - MainThread (8544):
Using wx version 4.1.1 msw (phoenix) wxWidgets 3.1.5 with six version 1.15.0
INFO - brailleInput.initialize (12:28:29.651) - MainThread (8544):
Braille input initialized
INFO - braille.initialize (12:28:29.654) - MainThread (8544):
Using liblouis version 3.17.0
INFO - braille.initialize (12:28:29.656) - MainThread (8544):
Using pySerial version 3.5
INFO - braille.BrailleHandler.setDisplayByName (12:28:29.661) - MainThread (8544):
Loaded braille display driver noBraille, current display has 0 cells.
INFO - core.main (12:28:29.856) - MainThread (8544):
Java Access Bridge support initialized
INFO - _UIAHandler.UIAHandler.MTAThreadFunc (12:28:29.876) - _UIAHandler.UIAHandler.MTAThread (6716):
UIAutomation: IUIAutomation6
INFO - core.main (12:28:30.176) - MainThread (8544):
NVDA initialized
ERROR - unhandled exception (12:31:02.820) - MainThread (8544):
Traceback (most recent call last):
  File "wx\core.pyc", line 3407, in <lambda>
  File "C:\Users\ntara\AppData\Roaming\nvda\addons\addonUpdater\globalPlugins\addonUpdater\addonHandlerEx.py", line 233, in _showAddonUpdateUICallback
    AddonUpdatesDialog(gui.mainFrame, info).Show()
  File "C:\Users\ntara\AppData\Roaming\nvda\addons\addonUpdater\globalPlugins\addonUpdater\addonGuiEx.py", line 116, in __init__
    self.Center(wx.BOTH | wx.CENTER_ON_SCREEN)
AttributeError: module 'wx' has no attribute 'CENTER_ON_SCREEN'

**Comment by zstanecic** [comment]
Hi Nikita,

This is the error of the NVDA addon you have installed on your system.

 

From: nikita ***@***.***> 
Sent: Thursday, April 22, 2021 11:50 AM
To: nvaccess/nvda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [nvaccess/nvda] errors in the last alpha. immediately after start-up (#12323)

 

hi. in the last alpha, an error in the nvda.exe file appears immediately after startup. nvda actually doesn't work. there were also errors in the log. INFO - main (12:28:28.821) - MainThread (8544):
Starting NVDA version alpha-22485,60c0b76b
INFO - core.main (12:28:28.914) - MainThread (8544):
Config dir: C:\Users\ntara\AppData\Roaming\nvda
INFO - config.ConfigManager._loadConfig (12:28:28.914) - MainThread (8544):
Loading config: C:\Users\ntara\AppData\Roaming\nvda\nvda.ini
INFO - core.main (12:28:29.266) - MainThread (8544):
Windows version: Windows 10 prerelease (10.0.21364) workstation
INFO - core.main (12:28:29.266) - MainThread (8544):
Using Python version 3.7.9 (tags/v3.7.9:13c94747c7, Aug 17 2020, 18:01:55) [MSC v.1900 32 bit (Intel)]
INFO - core.main (12:28:29.266) - MainThread (8544):
Using comtypes version 1.1.8
INFO - core.main (12:28:29.266) - MainThread (8544):
Using configobj version 5.1.0 with validate version 1.0.1
INFO - external:synthDrivers.RHVoice.SynthDriver.init (12:28:29.638) - MainThread (8544):
Using RHVoice version b'1.2.0'
INFO - synthDriverHandler.setSynth (12:28:29.651) - MainThread (8544):
Loaded synthDriver RHVoice
INFO - core.main (12:28:29.651) - MainThread (8544):
Using wx version 4.1.1 msw (phoenix) wxWidgets 3.1.5 with six version 1.15.0
INFO - brailleInput.initialize (12:28:29.651) - MainThread (8544):
Braille input initialized
INFO - braille.initialize (12:28:29.654) - MainThread (8544):
Using liblouis version 3.17.0
INFO - braille.initialize (12:28:29.656) - MainThread (8544):
Using pySerial version 3.5
INFO - braille.BrailleHandler.setDisplayByName (12:28:29.661) - MainThread (8544):
Loaded braille display driver noBraille, current display has 0 cells.
INFO - core.main (12:28:29.856) - MainThread (8544):
Java Access Bridge support initialized
INFO - _UIAHandler.UIAHandler.MTAThreadFunc (12:28:29.876) - _UIAHandler.UIAHandler.MTAThread (6716):
UIAutomation: IUIAutomation6
INFO - core.main (12:28:30.176) - MainThread (8544):
NVDA initialized
ERROR - unhandled exception (12:31:02.820) - MainThread (8544):
Traceback (most recent call last):
File "wx\core.pyc", line 3407, in 
File "C:\Users\ntara\AppData\Roaming\nvda\addons\addonUpdater\globalPlugins\addonUpdater\addonHandlerEx.py", line 233, in _showAddonUpdateUICallback
AddonUpdatesDialog(gui.mainFrame, info).Show()
File "C:\Users\ntara\AppData\Roaming\nvda\addons\addonUpdater\globalPlugins\addonUpdater\addonGuiEx.py", line 116, in init
self.Center(wx.BOTH | wx.CENTER_ON_SCREEN)
AttributeError: module 'wx' has no attribute 'CENTER_ON_SCREEN'

**Comment by XLTechie** [comment]
cc @josephsl
From the recent code removal.

**Comment by Nikita34196** [comment]
@zstanecic hi. by mistake in the log I realized that the error in the supplement. but by this there are errors in the files of the nvda itself. An error in the nvda.exe file appears immediately after startup. is this also related to additions? unfortunately, I can not accurately show this error because nvda on the last alpha due to the error almost does not work.

**Comment by feerrenrut** [comment]
This issue is with an addon which will need to be updated before the next release. I'm going to close this issue. You will have to either disable this addon (or all addons) or continue using an older version of NVDA until the addon is updated.
